src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -759,6 +759,24 @@ public void selectRecipientsToBeStudents(int qnNumber) {
         WebElement recipientDropdown = browser.driver.findElement(By.id("recipienttype-" + qnNumber));
         selectDropdownByVisibleValue(recipientDropdown, "Other students in the course");
     }
+    
+    public void enableOtherFeedbackPathOptions(int qnNumber) {
+        WebElement questionTable = browser.driver.findElement(By.id("questionTable" + qnNumber));
+        WebElement dropdownButton = questionTable.findElement(By.cssSelector(".feedback-path-dropdown > button"));
+        WebElement otherOption = questionTable.findElement(
+                                     By.className("feedback-path-dropdown-option-other"));
+        dropdownButton.click();
+        otherOption.click();
+    }
+
+    public void enableOtherFeedbackPathOptionsForNewQuestion() {
+        WebElement questionTable = browser.driver.findElement(By.id("questionTableNew"));
+        WebElement dropdownButton = questionTable.findElement(By.cssSelector(".feedback-path-dropdown > button"));
+        WebElement otherOption = questionTable.findElement(
+                                     By.className("feedback-path-dropdown-option-other"));
+        dropdownButton.click();
+        otherOption.click();
```

Comment:
I saw the Travis build report. All these `*.click()` must be `click(*)` as per the new version of Selenium's change in implementation.